packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -739,6 +739,8 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
       errors,
       eventData: {
         webClientDomain: window.location.hostname,
+        // @ts-ignore
+        isMercuryConnected: this.webex.internal.mercury?.connected,
```

Comment:
this kind of introduces a dependency of metrics plugin on the mercury plugin. It's not clear to me what the overall design for dependencies between all the SDK plugins is, but it feels like metrics should probably be lower layer than mercury and I can see that the mercury plugin has metrics plugin in the dependencies in the package.json, although I'm not sure what it uses it for. So this change would now make the dependencies kind of circular. Maybe it would be safer for mercury plugin to call some metrics API to tell it the status of the connection?